core/ts/__tests__/MaciState.test.ts

```diff
@@ -275,7 +275,7 @@ describe("MaciState", function () {
     it("should tally ballots correctly", () => {
       // Start with results = [0...0]
       let total = BigInt(0);
-      for (const v of poll.results) {
+      for (const v of maciState.polls[pollId].tallyResult) {
         total = total + v;
       }
```

Comment:
```suggestion
      const total = calculateTotal(maciState.polls[pollId].tallyResult);
```

Reply:
Given `calculateTotal()` going to be used only this code block for testing purpose, I'll use following code block instead:
```
// Check if all elements in tallyResult are 0
      const allZero = maciState.polls[pollId].tallyResult.every(v => v === BigInt(0));
      expect(allZero).to.eq(true);
```